.github/workflows/build-and-push.yml

```
@@ -0,0 +1,27 @@
+name: Build to release branches
+
+on:
+    workflow_dispatch:
+    push:
+        tags: ['*']
+        branches:
+            - dev-beta
+            - dev-main
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: false
+
+jobs:
+    build-assets:
+        uses: inpsyde/reusable-workflows/.github/workflows/build-and-push-assets.yml@add_option_to_select_built_branch_name
```

I merged https://github.com/inpsyde/reusable-workflows/pull/124, so we can use `@main`.
```suggestion
        uses: inpsyde/reusable-workflows/.github/workflows/build-and-push-assets.yml@main
```